blogs/data-mover/data-mover-1.2.0-overview.md

```diff
@@ -0,0 +1,134 @@
+
+<h2>What Is CSI?</h2>
+
+<p dir="auto">One of the more important components of Data Mover to understand is CSI, or Container Storage Interface. CSI provides a layer of abstraction between container orchestration tools and storage systems such that users do not need to be informed on the differences between storage provider's needs and requirements. It also provides point-in-time snapshotting of volumes.</p>
+
+<p dir="auto">CSI volumes are now the industry standard and are the storage backing for most Cloud Native applications. However, issues concerning CSI volumes still remain. Some volumes have vendor-specific requirements, and can prevent proper portability and durability. Data Mover works to solve this case, which will be discussed more in the next section.</p>
+
+<p dir="auto">You can read more about CSI<span>&nbsp;</span><a href="https://kubernetes-csi.github.io/docs/">here</a>.</p>
+
+<h2>Why We Need Data Mover</h2>
+
+<p dir="auto">During a backup using Velero with CSI, CSI snapshotting is performed. This snapshot is created on the storage provider where the snapshot was taken. This means that for some providers, such as ODF, the snapshot lives on the cluster. Due to this poor durability, in the case of a disaster scenario, the snapshot is also subjected to disaster.</p>
+
+<p dir="auto">With volumeSnapshotMover controller, snapshots are relocated off of the cluster to the targeted backupStorageLocation (generally object storage), providing additional safety.</p>
+
+<h2>OADP's Data Mover Roadmap</h2>
+<p dir="auto">The OADP team introduced the Data Mover feature in OADP-1.1.0 as a tech preview feature.  During the course of OADP-1.1 and OADP-1.2 development we have also been actively contributing to the upstream Velero project.  Our work in the upstream has focused on providing the foundation for the Data Mover feature directly in Velero.  We are pleased to see the upstream Velero project has officially adopted the feature and will release a <a href="https://github.com/vmware-tanzu/velero/wiki/1.12-Roadmap">built in Data Mover</a> in Velero 1.12. 
+The Data Mover feature in OADP-1.2 will remain in tech preview as we continue to contribute to and evalute Velero's built in Data Mover for our customers.
+<h2>Components</h2>
+
+<h3><a href="https://github.com/openshift/oadp-operator">OADP OPERATOR</a>:</h3>
+
+<p dir="auto">OADP is the OpenShift API for Data Protection operator. This open source operator sets up and installs Velero on the OpenShift platform, allowing users to backup and restore applications. We will be installing Velero alongside the CSI plugin (modified version).</p>
+
+<h3><a href="https://github.com/vmware-tanzu/velero-plugin-for-csi">CSI PLUGIN</a>:</h3>
+
+<p dir="auto">The collection of Velero plugins for snapshotting CSI backed PVCs using the CSI beta snapshot APIs.</p>
```

Comment:
VolumeSnapshot API graduated to v1 version: https://kubernetes.io/docs/concepts/storage/volume-snapshots/

Reply:
thanks!